### DISCUSSION TITLE
All I want is discovering new projects written in Delphi/Pascal, but that's not possible now.

### DISCUSSION THREAD

**Discussion by edwinyzh** [community/community#66632] [Other Feature Feedback, Questions, & Ideas]
### Issue about the new homepage feed

Product Feedback

### Body

I followed many people who has a interest in Delphi/Pascal so that I can get notified about their new project (most likely in Delphi/Pascal), about the project they've starred recently (most likely in Delphi/Pascal) in the github homepage.

In other words, all I want is discovering new projects written in Delphi/Pascal, but that's not possible now.

**Comment by brendanzab** [comment]
Yeah it really sucks how much the new feed completely squashes niche repositories :(

I used to be able to star a neat repo related to dependent types or something and have a bunch of people come along and star it in turn. It was a great way to learn about new and undiscovered things in your areas of interest. I struggle to see how that usage pattern could continue with this style of feed. We follow people for a reason!
